CODE_OF_CONDUCT.md

```diff
@@ -40,7 +40,7 @@ Project maintainers who do not follow or enforce the Code of Conduct in good fai
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4, available at [http://contributor-covenant.org/version/1/4][version]
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4, available at [https://www.contributor-covenant.org/version/1/4/code-of-conduct/][version]
 
-[homepage]: http://contributor-covenant.org
-[version]: http://contributor-covenant.org/version/1/4/
+[homepage]: https://www.contributor-covenant.org
```

Comment:
Why is it necessary to add the `www.` prefix? The old link works, and when I go to it, there's no www. in front.

Reply:
Yeah, the old link works.

It will redirect you to the `https://www.` version, maybe Chrome hide it from the URL bar again.

```
$ curl -LI  http://contributor-covenant.org
HTTP/1.1 301 Moved Permanently
Location: https://contributor-covenant.org/

HTTP/2 301
location: https://www.contributor-covenant.org/

HTTP/2 200
content-type: text/html; charset=UTF-8
```